integration/localstack/docker-compose.yml

```diff
@@ -11,7 +11,7 @@ services:
       - "127.0.0.1:53:53"
       - "127.0.0.1:53:53/udp"
       - "127.0.0.1:443:443"
-      - "127.0.0.1:4566:4566"
+      - "0.0.0.0:4566:4566"
```

Comment:
Not specific to this change, last I saw when using LocalStack, I thought that they were consolidating to a single port for their services endpoint. Do we need `4566` AND `4571`, or is 4571 for another service?

Reply:
I originally copied this from the docker-compose sample in the localstack gh https://github.com/localstack/localstack. I'm not sure what 4571 is used for.

Reply:
Finally found the old version of the [README](https://github.com/localstack/localstack/blob/2df3eca88ed0215f8d99ed05fd2d7ddb35636d34/README.md).

> Note: Starting with version 0.11.0, all APIs are exposed via a single edge service, which is accessible on http://localhost:4566/ by default

I was using LocalStack for a project back in 2020, shortly after they started migrating stuff to their edge port endpoint, so I recalled seeing 4571 (I think this is where their /health endpoint was hosted) when I was working with it, but moved everything to 4566 later.

Reply:
I can remove